pages/signup.tsx

```diff
@@ -1,3 +1,163 @@
+import Link from "next/link";
+import Head from "next/head";
+import "twin.macro";
+import { useForm } from "react-hook-form";
+import { UserPlus } from "phosphor-react";
+import {
+    Button,
+    ButtonProps,
+    Container,
+    Grid,
+    Text,
+    Heading,
+    Divider,
+    IconButton,
+    VStack,
+    HStack,
+    Image,
+} from "@chakra-ui/react";
+import { Box } from "@chakra-ui/core";
+
+import { useScreen } from "hooks";
+import { InputField } from "components/units/InputField";
+
+const data = [
+    { name: "Google", Icon: Image, source: "/images/google.png" },
```

Comment:
Will remove Icon objects from here since i ended up using image files for icons.

Reply:
Image is good but SVG is even better. NextJS doesn't support SVG by default but a way around this would be to have custom Webpack config with SVG plugin. Can you make a PR for this?